pkg/common/util/client.go

```diff
@@ -1,10 +1,26 @@
 package util
 
-import "os"
+import (
+	"os"
 
-func homeDir() string {
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func HomeDir() string {
 	if h := os.Getenv("HOME"); h != "" {
 		return h
 	}
 	return os.Getenv("USERPROFILE") // windows
 }
+
+// TODO (Jeffwan@): Find an elegant way to either use delegatingReader or directly use clientss
```

Comment:
```suggestion
// TODO (Jeffwan@): Find an elegant way to either use delegatingReader or directly use clients
```

Reply:
Nice catch. I will address all grammar issue in one PR.